bagit.xml

```diff
@@ -1,819 +1,800 @@
-three values, and any such characters in the URL &must; be percent-encoded
-<xref target="RFC3986"/>.  There is no limitation on the length of any
-of the fields in the "fetch.txt".
-</t>
-
-<t>
-The "fetch.txt" file allows a bag to be transmitted with
-"holes" in it, which can be practical for several reasons.  For example,
-it obviates the need for the sender to stage a large serialized copy of
-the content while the bag is transferred to the receiver.  Also, this
-method allows a sender to construct a bag from components that are either
-a subset of logically related components (e.g., the localized logical
-object could be much larger than what is intended for export) or
-assembled from logically distributed sources (e.g., the object components
-for export are not stored locally under one filesystem tree).
-</t>
-
-</section> <!-- Fetch File -->
-
-<section title="Other Tag Files" anchor="sec-other-tag-files">
-<t>
-A bag &may; contain other tag files that are not defined by this
-specification.
-Implementations &should; ignore the content of any unexpected tag files,
-except when they are listed in a tag manifest.
-When unexpected tag files are listed in a tag manifest, implementations
-&must; only treat the content of those tag files as octet streams for the
-purpose of checksum verification.
-</t>
-</section> <!-- /Other Tag Files -->
-</section> <!-- /Optional Elements -->
-
-<section title="Text Tag File Format" anchor="sec-tag-files">
-<t>
-All tag files specifically described in this specification &must; adhere to
-the text tag file format described below. Other tag files &may; adhere to 
-the text tag file format described below. 
-</t>
-<t>
-Text tag files are line-oriented, and each line &must; be
-terminated by a newline (LF), a carriage return (CR), or carriage return
-plus newline (CRLF).
-Text tag files &must; end in the extension ".txt".
-</t>
-
-<t>
-In all text tag files except for the bag declaration file, text &must; be 
-encoded in the character encoding specified in the "bagit.txt" bag declaration 
-file.  Text tag files except for the bag declaration file &may; include a
+As a result, no <spanx style="emph">filename</spanx> listed in a tag manifest begins "data/".
+</t>
+        </section>
+        <!-- /Tag Manifest -->
+        <section anchor="sec-bag-info" title="Bag Metadata: bag-info.txt">
+          <t>
+            The "bag-info.txt" file is a tag file that contains metadata
+            elements describing the bag and the payload. The metadata elements
+            contained in the "bag-info.txt" file are intended primarily for
+            human use. All metadata elements are optional and &may; be repeated.
+            Because "bag-info.txt" is intended for human reading
+            and editing, ordering &may; be significant and the ordering of
+            metadata elements &must; be preserved.
+          </t>
+          <t>
+            A metadata element &must; consist of a label, a colon, at least one
```

Comment:
Because of historical reasons. This version is just to take what we already have out of draft and make it an official RFC. There will be another version coming soon afterwards that will have many backwards breaking updates.